sensa/serviceconnector.py

```diff
@@ -323,7 +323,7 @@ def from_current_cli_profile(cls: Type[T], version: str = "3", **kwargs) -> T:
         :return: _description_
         :rtype: T
         """
-        cli_cfg = get_cortex_profile()
+        cli_cfg = get_sensa_profile()
         url, token = cli_cfg["url"], cli_cfg["token"]
         # type: ignore # ignore until mypy properyly supports attr ...
         return cls(url, version, token, **kwargs)
```

Comment:
I can't comment on the right line on GitHub, but a just below this we should probably rename `from_cortex_message()` -> `from_sensa_message()`..